docs/intro.txt

```diff
@@ -203,6 +203,13 @@ If partial results indicate that a job is taking longer run than anticipated the
 
 Also partial results enable users to quickly validate basic execution for very large jobs which may take days to complete.  
 
+Leakage Gadget Detection
+------------------------
+
+When running circuits on the :py:class:`QuantinuumBackend`, there is some small probability that a qubit will "leak" out of the qubit subspace leading to erroneous measurement results.
```

Comment:
When running circuits on the :py:class:`QuantinuumBackend`, one source of error is "leakage", where with some small probability a qubit will experience leakage into electronic states outside the qubit subspace. When this occurs, none of the remaining gates in the circuit will have any effect and so this leads to erroneous results.

Reply:
done